homeassistant/components/google_generative_ai_conversation/const.py

```diff
@@ -23,14 +24,15 @@
 {%- endfor %}
 """
 
+CONF_RECOMMENDED = "recommended"
+RECOMMENDED_LABEL = "Recommended (Gemini 1.5 Flash)"
 CONF_CHAT_MODEL = "chat_model"
-DEFAULT_CHAT_MODEL = "models/gemini-pro"
+RECOMMENDED_CHAT_MODEL = "models/gemini-1.5-flash-latest"
 CONF_TEMPERATURE = "temperature"
-DEFAULT_TEMPERATURE = 0.9
+RECOMMENDED_TEMPERATURE = 0.9
```

Comment:
Change default to 1 to match the default value of the flash:

```
Model(name='models/gemini-1.5-flash-latest',
      base_model_id='',
      version='001',
      display_name='Gemini 1.5 Flash',
      description='Fast and versatile multimodal model for scaling across diverse tasks',
      input_token_limit=1048576,
      output_token_limit=8192,
      supported_generation_methods=['generateContent', 'countTokens'],
      temperature=1.0,
      top_p=0.95,
      top_k=64)
```

Reply:
One more reason we should just configure this for the user :)